src/playlist.rs

```diff
@@ -1144,6 +1208,222 @@ impl DateRange {
     }
 }
 
+// Implementing structs for LL-HLS
+#[derive(Debug, Default, PartialEq, Clone)]
+pub struct ServerControl {
+    pub can_skip_until: Option<f64>,
+    pub hold_back: Option<f64>,
+    pub part_hold_back: Option<f64>,
+    pub can_block_reload: bool,
+}
+
+impl ServerControl {
+    pub(crate) fn from_hashmap(
+        mut attrs: HashMap<String, QuotedOrUnquoted>,
+    ) -> Result<ServerControl, String> {
+        let can_skip_until = unquoted_string_parse!(attrs, "CAN-SKIP-UNTIL", |s: &str| s
+            .parse::<f64>()
+            .map_err(|err| format!("Failed to parse CAN-SKIP-UNTIL attribute: {}", err)));
+        let hold_back = unquoted_string_parse!(attrs, "HOLD-BACK", |s: &str| s
```

Comment:
I'm also now seeing `CAN-SKIP-DATERANGES` under [4.4.3.8](https://author-tools.ietf.org/iddiff?url1=draft-pantos-hls-rfc8216bis-06&url2=draft-pantos-hls-rfc8216bis-07&difftype=--html) in the draft, could add that one as well